frontend/src/components/common/TooltipIcon.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import './TooltipIcon.scss';
 import { OverlayTrigger, Tooltip, Overlay } from 'react-bootstrap';
 import { FaRegQuestionCircle } from 'react-icons/fa';
 
 interface TooltipIconProps extends React.ComponentPropsWithRef<typeof Overlay> {
   toolTip?: string;
   toolTipId: string;
+  children?: ReactNode;
```

Comment:
Me too :(, tried this solution yesterday which I thought worked because `npm run-script build` worked locally but still gave me the error when I made a PR